gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/mapper/QuantityMapper.java

```diff
@@ -0,0 +1,58 @@
+package uk.nhs.adaptors.pss.translator.mapper;
+
+import java.util.List;
+
+import org.apache.commons.lang3.StringUtils;
+import org.hl7.fhir.dstu3.model.Quantity;
+import org.hl7.v3.PQR;
+import org.hl7.v3.Value;
+
+public class QuantityMapper {
+    private static final String UNIT_SYSTEM = "http://unitsofmeasure.org";
+    private static final String TYPE_IVL_PQ = "IVL_PQ";
+
+
+    public Quantity mapQuantity(Value value) {
```

Comment:
Could you try to separate some logic from this method to private methods, so it's a bit easier to read? Maybe for example create two separate methods that handle cases when `value.getHigh()` and `value.getLow()` aren't empty (just an idea)?